### ISSUE TITLE
Read with 3 parts break cutadapt with unexpected suffixes in the read ids

### ISSUE THREAD

**Issue by arteymix** [PavlidisLab/rnaseq-pipeline#60]
https://github.com/marcelm/cutadapt/issues/199

This is a problem for GSE171683.

The first read in SRR14182285 are mismatched due to the `.1` and `.3` suffixes:

```
@SRR14182285.1.1 HWI-ST1264:174:H5LVLBCX2:2:1101:1354:2225 length=100
CTCTCAAAAATTACAGGTTTAAATTATAACGTGTCCTTAAGTTTAGCATGGTTAAATGATATTTTCTTTTAGGGTAACCAAGAGCAGTTAGTTAAGTGAC
+SRR14182285.1.1 HWI-ST1264:174:H5LVLBCX2:2:1101:1354:2225 length=100
DDABBEHFFGG1CCFHIC@FFHHFHIHCHHHF@<?FCCCGHIIICGEH@<@FGC1FHIHHHIEHIHHHIICHEHHIIHEHIIEH1<GHHEHIHHIECHHE
```

```
@SRR14182285.1.3 HWI-ST1264:174:H5LVLBCX2:2:1101:1354:2225 length=101
CCGTTATTTTCGAGACAGAGTTTCTCTGTGTAGTCCTGGCTGTCCTAGAACTCACTCTGTAGACCAGACTGGCCTCAAACTCACAGAGATCTGCCTGCCTC
+SRR14182285.1.3 HWI-ST1264:174:H5LVLBCX2:2:1101:1354:2225 length=101
IID0DDB@F<</<C1F@ECFHEHH?1<D<1<DC1<1<<<CG1<CFHH@1<1<<F@<<1DGC1D<11<D<<<1<GHC1<1<DCG?<F@C<1D@CG@G?F@<1
```

This issue causes cutadapt to hang when run in parallel. This should be reported upstream.

**Comment by marcelm** [comment]
I’ve filed two issues for the two mentioned problems.

**Comment by arteymix** [comment]
We ended-up fixing this issue by removing `fastq-dump --readids` flag https://github.com/PavlidisLab/bioluigi/commit/79446c3a026eaceb31ddeaf77a07f794ff0df5d9. It will be nice to have at least the first thread hanging issue addressed upstream.

**Comment by arteymix** [comment]
I'll close this issue since we're not affected anymore.
